protobuf/api/execution.proto

```diff
@@ -19,37 +23,32 @@ service Execution {
 }
 
 // GetExecutionRequest defines request to retrieve a single execution.
+// **Example**	
+// TODO: add JSON example
 message GetExecutionRequest {
-  string hash = 1;
+  string hash = 1;  // Execution's hash to fetch.
 }
 
 // StreamExecutionRequest defines request to retrieve a stream of executions.
+// **Example**	
+// TODO: add JSON example
 message StreamExecutionRequest{
   // Filter contains filtering criteria.
   message Filter {
-    // Status for filtering execution by status.
-    // Note: to filter multiple statues or flags, eg:
-    // Failed | Completed
-    definition.Status status = 1;
-
-    // serviceHash is a hash for filtering execution by service hash.
-    string serviceHash = 2;
+    definition.Status status = 1; // Status to filter executions.
```

Comment:
I feel putting a comment in one line make it less readable. 
Also if the comment is a bit longer (like before) there is no option to make it readable at all.

Reply:
Nico explained to me about that, it's for 2 things:
- Not sure the generation of the documentation works with comments on top (but seems that it works https://github.com/pseudomuto/protoc-gen-doc#writing-documentation)
- It looks nicer when we have a list of attributes with everything aligned

Reply:
Still I think it's better to have clean docs (long and explaining in some cases) rather than aligned fields. And remember it's our docs for Public API! So it should be good and here I feel like it was shrink just to keep a small line length.

Reply:
ok guys!